src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -251,7 +251,7 @@ uct_component_t uct_rdmacm_component = {
     },
     .cm_config          = {
         .name           = "RDMA-CM connection manager",
-        .prefix         = "RDMACM_",
+        .prefix         = "RDMACM_CM_",
```

Comment:
why needed to change?

Reply:
so that it won't be the same as the env params from the previous rdmacm implementation (through iface, not cm)

Reply:
why not? does it create any env vars conflict in this PR?
we're going to remove old CM, so better keep shorter name

Reply:
I wanted to create a distinguish between the two flows.
Also, for tcp_sockcm it's TCP_CM. Shouldn't the sockaddr transports be the same in this aspect?

Reply:
so RDMA_CM_ ?

Reply:
Is it ok to split rdmacm..? 
RDMACM_CM isn't ok because of the duplication of CM in it?

Reply:
> RDMACM_CM isn't ok because of the duplication of CM in it?

Yes, IMO it doesn't look so good in user-facing param